src/mysql_sock_ssl.erl

```diff
@@ -0,0 +1,64 @@
+%% MySQL/OTP – MySQL client library for Erlang/OTP
+%% Copyright (C) 2014 Viktor Söderqvist
```

Comment:
Please use your own name and the current year for these new files. :-) Also feel free to add your own name to old files where major changes have been done. The year should probably be updated too.